cuda_core/cuda/core/experimental/_event.py

```diff
@@ -51,37 +82,45 @@ def _init(options: Optional[EventOptions]=None):
         return self
 
     def __del__(self):
+        """Return close(self)"""
         self.close()
 
     def close(self):
-        # Destroy the event.
+        """Destroys the event."""
```

Comment:
nit: the methods which return values are documented as "Return ..". It seems that "Destroy" is more alligned with that pattern.

Reply:
Done